larky/src/main/resources/vgs/http/request.star

```diff
@@ -19,6 +19,10 @@ def VGSHttpRequest(
         return self.full_url
     def _set_url(url):
         self.full_url = url
+
+        parsed_url = parse.urlsplit(url)
+        self.path = parsed_url.path
+        self.query_string = parsed_url.query
```

Comment:
Maybe if we are separating path and query string, let's provide vars for every other part of url?

Reply:
`urllib` takes care of generating `type` , `host` and `fragment` already, so we extended it so fill in the gap